src/engine/Instance.v3

```diff
@@ -79,7 +79,7 @@ class ExportedValueType(src: ExportedType, vt: ValueType) extends ExportedType {
 // An instantiated module, including memories, tables, functions, etc.
 class Instance(module: Module, imports: Array<Exportable>) {
 	def memories = Array<Memory>.new(module.memories.length);
-	def functions = Array<Function>.new(module.functions.length);
+	var functions = Array<Function>.new(module.functions.length);
```

Comment:
The prior function in `wizeng.new_funcref` doesn't append to the instance. I think we can get by without appending for now. That means that these functions can be garbage-collected.